safe/api.py

```diff
@@ -164,6 +164,18 @@ def commit(self):
     def session(self):
         return self.api.session
 
+    @property
+    def version(self):
+        return self.api.version
+
+    def findpath(self, path):
+        node = self
+        for attr in path.split('.'):
```

Comment:
This looks a lot like [`operator.attrgetter`](https://docs.python.org/2/library/operator.html#operator.attrgetter) although I guess your code returns the longest path even if some child elements can't be found? Not sure if that's desired behavior or not?